Dockerfile

```
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04 as build
+ENV TZ=UTC
+
+LABEL Cammoguy <jpburnett@gwu.edu>
+
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && \
+    apt-get update && apt-get install -y \
+    build-essential \
+    binutils-mips-linux-gnu \
+    pkg-config \
+    python3 \
+    python3-pip \
+    git \
+    wget \
+    unzip \
```

```suggestion
    unzip \
    vbindiff \
    vim \
```
vbindiff is so important when you aren't sure what isn't matching, Vim I want because of a script of mine that dumps details into a editable window.